### PR TITLE
integrate zcash local net test fixtures into zaino tests

### PR DESCRIPTION
**Pull request by Oscar-Pepper**

### CODE REVIEW

**Comment by idky137** [comment]
- Work towards https://github.com/zingolabs/zaino/issues/51

**Review by idky137 — CHANGES_REQUESTED** [review]
Code all looks good. Im getting errors running tests though. Is there anything I need to do apart from build release? Could you add to the Testing section of the readme?

**Review by idky137 — CHANGES_REQUESTED** [review]
Code all looks good. Im getting errors running tests though. Is there anything I need to do apart from build release? Could you add to the Testing section of the readme?

**Comment by Oscar-Pepper** [comment]
> Code all looks good. Im getting errors running tests though. Is there anything I need to do apart from build release? Could you add to the Testing section of the readme?

ok i have added to the readme. I am ignoring mempool tests until https://github.com/zingolabs/zcash-local-net/pull/19 merges

**Comment by Oscar-Pepper** [comment]
i have unignored mempool tests and pointed at my branch for now. we can clean this up later when zcash_local_net is up to date

File: README.md [hunk 1]
```diff
@@ -54,8 +54,14 @@ Holds tonic generated code for the lightwallet service RPCs and compact formats.
...
 - To run tests:
-1) Simlink or copy compiled `zcashd`, `zcash-cli` and `lightwalletd` binaries to `$ zingo-indexer/zingo-testutils/test_binaries/bins/*`
-3) Run `$ cargo nextest run` or `$ cargo test`
+1) Simlink or copy compiled `zcashd`, `zcash-cli` and `lightwalletd` binaries to `$ zaino/test_binaries/bins/*`
+2) Run `$ cargo nextest run` or `$ cargo test`
+
+- To run client rpc tests:
+1) Simlink or copy compiled `zcashd` and `zcash-cli` binaries to `$ zaino/test_binaries/bins/*`
+2) Build release binary `cargo build --release` WARNING: these tests do not use the binary built by cargo nextest
+3) Generate the chain cache `cargo nextest run generate_zcashd_large_chain_cach --run-ignored ignored-only --features test_fixtures`
```
Comment (idky137) [inline README.md]: This is returning an error:

$ cargo nextest run generate_zcashd_large_chain_cach --run-ignored ignored-only --features test_fixtures
error: none of the selected packages contains these features: test_fixtures

Does this feature need to be added to zaino?

Edit: line 63 should be:

3) Generate the chain cache `cargo nextest run generate_zcashd_chain_cach --run-ignored ignored-only --features test_fixtures`

**Review by idky137 — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (idky137) [reply README.md]: fixed in 65d5d6f.

**Review by idky137 — APPROVED** [review]

**Pull request merged**